dbengine/db.cc

```diff
@@ -43,12 +44,12 @@ opStatus database::readValueFromKey(std::string key,
 
 opStatus database::readAllValues(int clientSocket) {
   opStatus memtableStatus = memtableWrapperObjRef->getAllValues(clientSocket);
-  if(memtableStatus != opStatus::opSuccess) {
-    return memtableStatus;
-  }
```

Comment:
Why delete this?
if(memtableStatus != opStatus::opSuccess) {
  sendEndMsgToClient(clientSocket);
  return memtableStatus;
}